src/test/java/selenium/tutorial/studies/RightClickTest.java

```diff
@@ -0,0 +1,59 @@
+package selenium.tutorial.studies;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import junit.framework.Assert;
+
+public class RightClickTest {
+	
+	WebDriver driver;
+	
+	@BeforeTest
+	public void setup() throws IOException {
+
+		InputStream input = RightClickTest.class.getClassLoader().getResourceAsStream("testData.properties");
+		Properties property = new Properties();
+		property.load(input);
+		
+		if("chrome".equals(property.getProperty("activeBrowser"))) {
+			driver=new ChromeDriver();
+			driver.manage().window().maximize();
```

Comment:
same code is written in both if and else  blocks. better put it in a common place so that both drivers can execute

Reply:
same code is written in both if and else  blocks. better put it in a common place so that both drivers can execute.